hypha/templates/includes/user_menu.html

```diff
@@ -1,67 +1,75 @@
 {% load i18n heroicons %}
 
 {% if request.user.is_authenticated %}
-    <div x-data="{ show: false }" class="relative flex">
+    <div x-data="{ show: false }" x-on:keydown.escape="show = false" class="relative flex">
         <button
             x-on:click="show = ! show"
-            class="font-bold flex items-center ps-1 pe-4 hover:bg-black hover:text-white transition-colors rounded-sm"
+            class="font-bold flex items-center transition-all rounded-sm px-1.5 bg-gray-100"
```

Comment:
```suggestion
            class="font-bold flex items-center transition-all rounded-sm px-1.5 group"
```

For me the button slightly stands out against the background in a way that doesn't feel intentional, almost like it's a hover state
<img width="261" alt="Screenshot 2024-07-03 at 12 19 04" src="https://github.com/HyphaApp/hypha/assets/145372368/04680372-7722-4614-b744-d16726787448">

Added `group` for the comment below on the user icon